pkg/setting/setting.go

```diff
@@ -661,73 +661,264 @@ func (cfg *Cfg) Load(args *CommandLineArgs) error {
 		return err
 	}
 	cfg.ProvisioningPath = makeAbsolute(provisioning, HomePath)
-	server := iniFile.Section("server")
-	AppUrl, AppSubUrl, err = parseAppUrlAndSubUrl(server)
+	if err := readServerSettings(iniFile, cfg); err != nil {
+		return err
```

Comment:
Nit: Maybe return some context when returning errors from reading certain sections/settings by wrapping error in "failed to read <section/setting> settings" or similar?

Reply:
@marefr Isn't that out of scope for the PR, since all it does is refactor?

Reply:
Sure